Environments/SharedServices/parameters.json

```diff
@@ -1,10 +1,12 @@
 {
-    "Organization": "file(../_Common/organizationName.txt)",
+    "Organization": "env(ORGANIZATION_NAME)",
     "DeploymentName": "shrdsvcs",
     "InstanceName": "${Parameters.Organization}-${Parameters.DeploymentName}",
     "Subscription": "SharedServices",
     "ModuleConfigurationParameters": {
         "DeploymentUserId": "env(DEPLOYMENT_USER_ID)",
```

Comment:
Yes, since we are using SPN to deploy the resources, this value is the id of the SPN Object ID, DeploymentUserId might need to be renamed, to ManagementUserId or something similar, this Id is meant to be the Id of the person logging into the Portal to manage Key Vault.